app/v2/notifications/post_notifications.py

```diff
@@ -222,3 +282,27 @@ def get_reply_to_text(notification_type, form, template):
         reply_to = template.get_reply_to_text()
 
     return reply_to
+
+
+def get_precompiled_letter_template(service_id):
+    template = Template.query.filter_by(
```

Comment:
Could make this into a dao method as I'll need something for determining if a letter is precompiled or not